frontend/src/lib/components/IframedToolbarBrowser/iframedToolbarBrowserLogic.ts

```diff
@@ -255,6 +279,9 @@ export const iframedToolbarBrowserLogic = kea<iframedToolbarBrowserLogicType>([
                         actions.setNewActionName(null)
                         actions.disableElementSelector()
                         return
+                    case PostHogAppToolbarEvent.PH_TOOLBAR_NAVIGATED:
+                        // remove leading / from path
+                        return actions.setCurrentPath(payload.path.replace(/^\/+/, ''))
```

Comment:
This function returns void so technically you should be doing something like this:
```suggestion
                        actions.setCurrentPath(payload.path.replace(/^\/+/, ''))
                        return void
```